backend/Makefile

```
@@ -53,13 +60,18 @@ dump-data:
 	@CMD="sed -E -i 's/\"email\": *\"([^\"]|\\\")*\"/\"email\": \"\"/g' data/nest.json" $(MAKE) exec-backend-command
 	@CMD="gzip -f data/nest.json" $(MAKE) exec-backend-command
 
+dump-db-data-e2e:
```

Use consistent naming
```suggestion
dump-data-e2e:
```